packages/react-vapor/rollup.config.js

```diff
@@ -75,68 +75,81 @@ export default {
         'redux',
         'underscore',
         'coveo-styleguide',
-        'underscore.string',
+        'underscore.string'
     ],
+    globals: {
+        codemirror: 'CodeMirror',
+        d3: 'd3',
+        jquery: '$',
+        react: 'React',
+        'react-dom': 'ReactDOM',
+        'react-redux': 'ReactRedux',
+        redux: 'Redux',
+        underscore: '_',
+        'coveo-styleguide': 'VaporSVG',
+        'underscore.string': 's',
+        'react-dom/server': 'ReactDOMServer'
+    },
     onwarn,
     plugins: [
         inject({
-            jQuery: 'jquery', // chosen-js expects jQuery to be available as a global
+            jQuery: 'jquery' // chosen-js expects jQuery to be available as a global
         }),
         replacePlugin(),
         postcss({
             extract: false,
             modules: {localIdentName: '[name]-[local]-[hash:base64]'},
             namedExports: true,
-            use: ['sass'],
+            use: ['sass']
         }),
         scssVariable(),
         resolve({
-            browser: true,
+            browser: true
         }),
         commonjs({
             namedExports: {
                 'hogan.js': ['Template', 'compile'],
                 'react-modal': ['setAppElement'],
                 'react-dnd': ['DragDropContext', 'DropTarget', 'DragSource'],
-                diff2html: ['diffChars', 'diffWordsWithSpace'],
-            },
+                diff2html: ['diffChars', 'diffWordsWithSpace']
+            }
         }),
-        tsPlugin(),
-    ],
+        tsPlugin()
+    ]
 };
 
 function tsPlugin() {
     return typescript({
         transformers: [
-            (service) => ({
+            service => ({
```

Comment:
When @GermainBergeron makes a PR, it adds the parentheses back, and you edit the the file it removes them 😄

Reply:
🙈